packages/flutter_tools/lib/src/commands/build_aot.dart

```diff
@@ -38,6 +38,10 @@ class BuildAotCommand extends BuildSubCommand with TargetPlatformBasedDevelopmen
         defaultsTo: false,
         help: 'Report timing information about build steps in machine readable form,',
       )
+      ..addFlag('track-widget-creation',
```

Comment:
We should comment somewhere that the flag is only exposed for simplicity writing build scripts and not because we ever plan on supporting this flag on AOT. We could support on AOT but that would be bad as it would bloat AOT code output size without a benefit.

Reply:
Done